Remote-Control/Remote-Control-Web/README.md

```diff
@@ -1,51 +1,57 @@
 # Agora Web Remote Control Demo
 
-这是一个利用 AgoraRTCSDK 和 AgoraRTMSDK 实现浏览器远程控制桌面的一个 Demo，以下简称为 AgoraWRC。
+*English | [中文](README_zh.md)*
 
-## 使用说明
-本 Demo 分为 2 个部分：基于 `Electron` 的桌面端和基于 `Web` 的网页端。其中，桌面端是受控端，负责发布自己的屏幕共享流以及执行来自网页端的控制命令，网页端是控制端，负责订阅桌面端的屏幕共享流，并根据鼠标键盘事件向桌面端发送控制命令。
+This is a web based remote control demo, built on `AgoraRTCSDK` and  `AgoraRTMSDK`.
 
-使用时，需要先打开桌面端的应用程序，选择 `作为受控设备`。待连接建立完成后，界面上会给出当前 AgoraWRC 运行的 `channel` 和 `password`。之后再在 **另一台** 设备上打开网页端的网页，先后输入相应的 `channel` 和 `password` 即可远程控制刚刚的设备。
+## How to use
 
-> 目前只支持一对一的远程控制
+This demo has 2 parts: 
 
-## 构建及代码说明
-首先，整个代码分为桌面端和网页端 2 个部分，也就是当前目录下的 `wrc-desktop` 和 `wrc-website` 文件夹，我们之后会分开介绍。其次，整个代码是基于 `AgoraRTC` 和 `AgoraRTM` 2 个 SDK，在构建代码之前，需要您手动将您的 `APPID` 同步到代码里。
+	- The desktop app based on `Electron` 
+	- The website app based on `ReactJS`
 
-> 因为考虑到泛用性，Demo 在编写的时候默认您没有开启 [App 动态密钥](https://docs.agora.io/cn/Agora%20Platform/terms?platform=All%20Platforms&#app-certificate), 是直接通过 APPID 加入 channel 的，如果需要启用动态密钥，您可能需要手动修改部分代码
+The desktop app is the controlled end, which will publish its screen-sharing stream and execute control signals from the website app. The website app is the controller end, which will subscribe the remote screen-sharing sharing and sends control signals based on captured mouse/keyboard events.
 
-接下来开始正式的代码说明和构建步骤
+When using, you need to open the desktop app first and click  `As the controlled device`. After the connection is established, the app will show your current `channel` and `password`. Then open the website app on **another device** and enter the current  `channel` and `password`.
 
-### 网页端：wrc-website
+> This demo only supports one to one remote control
 
-先看网页端，这是通过 `create-react-app` 生成的项目，使用的技术栈是 `react` + `mobx`，下面介绍一下大致的代码构成:
+## How to build
 
-- `./src/pages` 这里主要存放不同的路由页面 UI，主要就 2 个，主页和远程控制页
-- `./src/controller` 这里主要存放 AgoraWRC 的核心控制逻辑，AgoraRTC 和 AgoraRTM 2 个 SDK 主要的操作都在这里，主要是订阅远端流以及捕获鼠标/键盘事件并通过 RTM 广播
-- `./src/stores` 这里存放 Mobx Store，用来管理 UI 相关的状态，以及处理一些通过 UI 触发的 action
-  - `./src/stores/wrcStore.ts` 这是用来处理和传递 AgoraWRC 相关逻辑的核心 Store，差不多算是 AgoraWRC Controller 和 UI 的中间层
-- `./src/types` 因为网页端和桌面端会有信令交换，这里定义了信令的基本格式和类型
+At first, this demo's code has 2 parts (the desktop and the website), which are the `wrc-desktop` and `wrc-website` folders in the current directory, so you need to build 2 projects. Second, this demo is base on `AgoraRTCSDK` and `AgoraRTMSDK`, so you need to manually update your `APPID` into the code before building the code.
```

Comment:
- `wrc-desktop` and `wrc-website` are two different projects and need to be built respectively.
- you will need to manually update your `APPID` into the code before building the code.